.github/workflows/multinode.yml

```diff
@@ -0,0 +1,364 @@
+---
+# This reusable workflow deploys a multi-node test cluster on a cloud using
+# Terraform, then deploys OpenStack via Kayobe. Tempest is then used to test
+# the cloud.
+
+name: Multinode
+
+on:
+  workflow_call:
+    inputs:
+      multinode_name:
+        description: Multinode cluster name
+        type: string
+        required: true
+      multinode_controller_count:
+        description: Controller count
+        type: number
+        default: 3
+      multinode_compute_count:
+        description: Compute count
+        type: number
+        default: 2
+      multinode_storage_count:
+        description: Storage count
+        type: number
+        default: 3
+      os_distribution:
+        description: Host OS distribution
+        type: string
+        default: rocky
+      os_release:
+        description: Host OS release
+        type: string
+        default: '9'
+      ssh_username:
+        description: User for terraform to access the all-in-one VM
+        type: string
+        default: cloud-user
+      neutron_plugin:
+        description: Neutron ML2 plugin
+        type: string
+        default: ovn
+      stackhpc_kayobe_config_version:
+        description: stackhpc-kayobe-config version
+        type: string
+      stackhpc_kayobe_config_previous_version:
+        description: stackhpc-kayobe-config previous version
+        type: string
+      terraform_kayobe_multinode_version:
+        description: terraform-kayobe-multinode version
+        type: string
+        default: main
+      upgrade:
+        description: Whether to perform an upgrade
+        type: boolean
+        default: false
+      break_on:
```

Comment:
might be better as a choice rather than a string

Reply:
I've now seen that the workflow calling this one has a choice, which makes sense